src/decorators/validated-string-api-property.decorator.ts

```diff
@@ -27,13 +29,19 @@ export const ValidatedStringApiProperty = ({
 }: Options) => {
   minLength = length ?? minLength ?? 0;
   maxLength = length ?? maxLength;
+
+  const { shouldPropertyBeDocumentedAsRequired, validationDecoratorsToApply } = parseRequiredAndNullable({
+    required,
+    nullable: true,
```

Comment:
To match the behaviour we had before,we should have nullable being the logical negation of required, not always `true`

Reply:
We should also add API tests for this

Reply:
Amazing spot! I've made the fix and updated the API tests to cover this next time